controllers/storagecluster/reconcile.go

```diff
@@ -853,6 +853,12 @@ func validateCustomStorageClassNames(sc *ocsv1.StorageCluster) error {
 		}
 		scMap[sc.Spec.Encryption.StorageClassName] = true
 	}
+	if sc.Spec.Encryption.CephFs.StorageClass && sc.Spec.Encryption.KeyManagementService.Enable && sc.Spec.Encryption.CephFs.StorageClassName != "" {
+		if _, ok := scMap[sc.Spec.Encryption.CephFs.StorageClassName]; ok {
+			duplicateNames = append(duplicateNames, "Encryption")
```

Comment:
how about this ?
(and for RBD, append "RBD Encryption"... instead of just using "Encryption" in both cases)
```suggestion
			duplicateNames = append(duplicateNames, "CephFS Encryption")
```

Reply:
yeah correct, changed it